BuildAndTest.cmd

```diff
@@ -19,18 +19,20 @@ if "%ERRORLEVEL%" NEQ "0" (
 )
 popd
 
-
 pushd .\src\security_utilities_rust_ffi\
 call cargo clean --release
-call cargo build --release
+call cargo build --release --target x86_64-pc-windows-msvc
+call cargo build --release --target i686-pc-windows-msvc
 if "%ERRORLEVEL%" NEQ "0" (
   echo "security_utilities_rust_ffi build failed..."
   exit /b %ERRORLEVEL%
 )
 popd
 
-xcopy /Y .\src\security_utilities_rust_ffi\target\release\microsoft_security_utilities_core.dll .\refs
-xcopy /Y .\src\security_utilities_rust_ffi\target\release\microsoft_security_utilities_core.pdb .\refs
+xcopy /Y .\src\security_utilities_rust_ffi\target\i686-pc-windows-msvc\release\microsoft_security_utilities_core.dll .\refs\win-x86
+xcopy /Y .\src\security_utilities_rust_ffi\target\i686-pc-windows-msvc\release\microsoft_security_utilities_core.pdb .\refs\win-x86
```

Comment:
>pdb [](http://example.com/codeflow?start=113&length=3)

i don't see your checked in pdbs. pdbs won't get added automatically via git.exe you need to add the --force arg to get er done.

Reply:
Fixed.